source/scss/_pf-component-alert.scss

```diff
@@ -0,0 +1,38 @@
+//
+// Alerts
+// --------------------------------------------------
+
+.alert {
+  padding-left: $spacer;
+  padding-right: $spacer;
+  display: flex;
+  align-items: center;
```

Comment:
I'd say this should be `flex-start` since we want them align to the top when the text wrap into more than one line.